grpc-mqtt.cabal

```diff
@@ -62,7 +62,7 @@ common common
     , proto3-wire          >= 1.2.2    && < 1.5
     , relude               >= 0.7.0    && < 1.3
     , stm                  >= 2.5.0    && < 2.6
-    , text                 >= 0.2      && < 2.2
+    , text                 >= 0.2      && < 2.3
```

Comment:
`2.3` is not a version of the [`text`](https://hackage.haskell.org/package/text) package. I think this should probably be `2.2`?